script/ablist.yml

```
@@ -85,6 +85,24 @@
     - nuts
     - mental case
     - mental
+- type: simple
+  note: When describing a mathematical or programmatic value, using the word "sane" needlessly invokes the topic of mental health.  Consider using a domain-specific or neutral term instead.
```

```suggestion
  note: When describing a mathematical or programmatic value, using the word “sane” needlessly invokes the topic of mental health.  Consider using a domain-specific or neutral term instead.
```

Let’s use curly quotes here!